app/assets/stylesheets/application.scss

```diff
@@ -413,6 +413,11 @@ td.keys {
         text-overflow: ellipsis;
       }
     }
+    .notification-subject-author:before {
+      content: "by ";
+      color: $text-muted;
+      font-size: 10px;
+    }
```

Comment:
I can't CSS, so like...feel free to throw tomatoes. 🍅  I just think the stacked nature of the repo/author on mobile looked kinda weird and this normalizes it a little bit.